### DISCUSSION TITLE
How does vitepress support RTL language direction ?

### DISCUSSION THREAD

**Discussion by moHaHa** [vuejs/vitepress#2336] [General]
**Why I'm asking this question ?** 

because when I take a look at one of the [vuejs-docs](https://github.com/vuejs/docs) forks which is [vuejs-docs-ar](https://github.com/Abdelaziz18003/vuejs-docs-ar)  for translate Vuejs docs to [Arabic language](https://en.wikipedia.org/wiki/Arabic) , I visit the website the preview the Arabic transition  [vuejs-docs-ar.netlify.app](https://vuejs-docs-ar.netlify.app/) which has done 80% of the must important pages [vuejs-docs-ar | translation tasks ](https://github.com/Abdelaziz18003/vuejs-docs-ar/issues/2) 

as we all now that vuejs-decs based on vitepress, I opend vitepress docs to see how can build RTL language documentation
all I found is this 

[rtl-support-experimental](https://vitepress.dev/guide/i18n#rtl-support-experimental)
 
> ### RTL Support (Experimental)
>For RTL support, specify dir: 'rtl' in config and use some RTLCSS PostCSS plugin like https://github.com/MohammadYounes/rtlcss, https://github.com/vkalinichev/postcss-rtl or https://github.com/elchininet/postcss-rtlcss. You'll need to configure your PostCSS plugin to use :where([dir="ltr"]) and :where([dir="rtl"]) as prefixes to prevent CSS specificity issues.

this section is ununderstandable and  not enough to guide someone init RTL feature vitepress docs , for some one like me or even  vuejs-docs-ar repo team, becaouse the make overrides.css file to make the minimum require to read  arabic content in  [vuejs-docs-ar.netlify.app](https://vuejs-docs-ar.netlify.app/)  , this still not enough (still layout has rtl issue)

 I think an issue like this should be fixed in vitepress workspace, not vuejs-docs-ar translation in order to feature it in vitepress officail , so  we need to discuss how Vitepress will support RTL language direction 
 I mean by "we" (me and anyone interested, vitepress devs, more people familiar with RTL websites ), which way of support RTL, becaouse the wrong ways can cause many issues on parts of the project

**Comment by Pezhvak** [comment]
Any updates on this?

**Comment by pepicrft** [comment]
I'm interested in this one too for translating the [Tuist documentation](https://docs.tuist.dev)
